src/Time.roc

```diff
@@ -0,0 +1,12 @@
+interface Time
+    exposes [
+        now,
+    ]
+    imports [Effect, InternalTask, Task.{ Task }]
+
+## Milliseconds since UNIX EPOCH
+now : Task U128 *
```

Comment:
Cool! I've thought about time designs for a bit, and here's an idea:

* Rename this module from `Time` to `Utc`
* Have it expose an opaque type `Utc := U128`
* Have this function be `now : Task Utc *` instead
* Add `Utc.toMsSinceEpoch : Utc -> U128`

The nice thing about this design is that it means in the future we can add things like:
* nanosecond resolution instead of millisecond resolution
* deltas between two UTC times
* ways to convert from UTC to local time zones

What do you think?

Reply:
Done 👍